Redmine 25106 FO help text in case of missing appointment not configurable

ManssourBlh opened the pull request:
https://dev.lutece.paris.fr/bugtracker/issues/25106

ryahiaoui commented:
Could you please make a clean pull request without merge commits and with the correct syntax?
"LUT-25109: message"

ManssourBlh commented:
> Could you please make a clean pull request without merge commits and with the correct syntax? "LUT-25109: message"

Hi @ryahiaoui, I've updated the pull request as requested. I've removed the merge commits. Additionally, I've amended the commit message to include the correct syntax with the issue key 'LUT-25109'. Please review the updated pull request at your convenience. Thank you!

ryahiaoui commented:
You didn't consider my second remark about the formMessages.getNoAvailableSlot() statement when it returns empty

ManssourBlh commented:
> You didn't consider my second remark about the formMessages.getNoAvailableSlot() statement when it returns empty

Hello, thank you for reviewing my PR. However, I don't see any remark related to the formMessages.getNoAvailableSlot() statement in the conversation. However, I will go ahead and address this issue in my code.

ManssourBlh commented on src/java/fr/paris/lutece/plugins/appointment/web/AppointmentApp.java:
```diff
@@ -473,9 +471,10 @@ public synchronized XPage getViewAppointmentCalendar( HttpServletRequest request
                 }
             }
-            if ( firstDateOfFreeOpenSlot == null )
+            if ( firstDateOfFreeOpenSlot == null && formMessages != null
```
I apologize for not seeing your comment earlier. It appears that I was not receiving notifications from GitHub. I will  fix my notification settings so that I can respond to feedback more promptly in the future. Thank you for bringing this to my attention.

ryahiaoui commented:
Thanks
